suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -173,12 +169,30 @@ export const useAddCoinAccount = () => {
             },
         });
 
-    const showAccountTypeBottomSheetWithDefaultAccount = (network: Network) => {
+    const showAddAccountGeneralErrorAlert = () =>
```

Comment:
nit: Could be just `showGeneralErrorAlert` for better readibility.
```suggestion
    const showGeneralErrorAlert = () =>
```

The `AddAccount` is obvious in context of this file.

Reply:
868b50a92f